README.md

```diff
@@ -23,32 +23,25 @@ First you must annotate your Kotlin data classes with the `@JsonSerializable` an
 data class Person(
     val name: String,
     val email: String?,
-    // This property uses a custom getter name which requires two annotations.
-    @get:JvmName("hasVerifiedAccount") @Getter("hasVerifiedAccount")
     val hasVerifiedAccount: Boolean,
     // This property has a different name in the Json than here so @Json must be applied.
     @Json(name = "created_at")
     val signUpDate: Date,
-    // This field has a json qualifier applied, the generated adapter will request an adapter with the qualifier.
-    @NullIfEmpty
-    val jobTitle: String?
+    // This field has a default value which will be used if the field is missing.
+    val jobTitle: String? = null
 )
 ```
 
 Then create a class that will be your factory:
 ```kotlin
 @KotshiJsonAdapterFactory
-abstract class ApplicationJsonAdapterFactory : JsonAdapter.Factory {
-    companion object {
-        val INSTANCE: ApplicationJsonAdapterFactory = KotshiApplicationJsonAdapterFactory()
-    }
-}
+object ApplicationJsonAdapterFactory : JsonAdapter.Factory by KotshiApplicationJsonAdapterFactory()
```

Comment:
This seem to be a bit more stable than last time I tried it, it might still need to revert to the ugly abstract class solution